fixtures/etc/fog.json

```
@@ -0,0 +1,36 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "fog",
```

```suggestion
  "name": "F-100",
```

Question: if ETC bought High-End, do we need a separate manufacturer for High-End, or do all of their fixtures get folded into ETC now?